sys/linux/bpf.txt

```diff
@@ -400,19 +408,25 @@ bpf_obj_get {
 	file_flags	flags[bpf_open_flags, int32]
 }
 
+bpf_attach_targets [
+	zero	const[0, int32]
+	cgroup	fd_cgroup
+	map	fd_bpf_map
+] [size[32]]
```

Comment:
This size is in bytes. I assume we want 4 here?
But if we leave just 2 fd's here, this becomes not too useful -- all fd's are 4 bytes anyway.

Reply:
Yes, I wanted 4 bytes.